astropy/coordinates/polarization.py

```diff
@@ -0,0 +1,255 @@
+from collections import namedtuple
+from contextlib import contextmanager
+from typing import Dict
+
+import numpy as np
+
+from astropy.utils import unbroadcast
+from astropy.utils.data_info import MixinInfo
+from astropy.utils.shapes import ShapedLikeNDArray
+
+__all__ = ["StokesCoord", "custom_stokes_symbol_mapping", "StokesSymbol"]
+
+
+StokesSymbol = namedtuple("StokesSymbol", ["symbol", "description"], defaults=[""])
```

Comment:
An equivalent, but [more pythonic](https://stackoverflow.com/a/50767206) version of namedtuple

```suggestion
class StokesSymbol(NamedTuple):
    """Symbol for a Stokes coordinate."""

    symbol: str = ""
    description: str = ""
```